### ISSUE TITLE
Chaining animations

### ISSUE THREAD

**Issue by kitab15** [MengTo/Spring#123]
I have a SpringButton and I want to create a composed animation. 
Which is the proper way? 
How I use the methods below to compose an animation?
```
animate()
animateNext { ... }
animateTo()
animateToNext { ... }
```

Thanks in advance.

**Comment by royherma** [comment]
yes, would appreciate some more documentation on chaining and using these methods. The demo app has examples, but only covers the animate() method

**Comment by acegreen** [comment]
Not sure if anyone is still interested, but here is a chain

        rewardsButton.animation = "fadeIn"
        rewardsButton.animateNext {
            self.rewardsButton.animation = "shake"
            self.rewardsButton.repeatCount = 1000
            self.rewardsButton.animate()
        }

So I wanted to fade in then shake my button. Would be nice to have infinite repeatCount

**Comment by calingo** [comment]
I want to make a Label fade in, wait a 1 sec, fade out, then another Label to do the same and another. Ideally with option to control the timing between each fade. Is that something I can do with Spring?

**Comment by francisykl** [comment]
+1 on this, having an infinite repeat count would be nice.
